docs/install-config/configure-yml-file.md

```diff
@@ -245,7 +240,7 @@ The following table lists the additional, optional parameters that you can set t
   <tr>
     <td valign="top"><code>external_database</code></td>
     <td valign="top">&nbsp;</td>
-    <td valign="top">Configure external database settings, if you disable the local database option. Currently, Harbor only supports PostgreSQL database. You must create four databases for Harbor core, Clair, Notary server, and Notary signer. The tables are generated automatically when Harbor starts up.</td>
+    <td valign="top">Configure external database settings, if you disable the local database option. Currently, Harbor only supports PostgreSQL database. You must create three databases for Harbor core, Notary server, and Notary signer. The tables are generated automatically when Harbor starts up.</td>
```

Comment:
three separate databases? or tables?

Reply:
i assumed databases, since that is what is said before. should it be tables? Re-reading this, it makes more sense to say databases here, since the next section says "tables are generated automatically when Harbor starts up", why would you need to create the 3 tables if they are generated automatically when Harbor starts?


Would a user still need to create let harbor know about an external Clair database to use Clair as an external scanner?

Reply:
lgtm